modin/core/dataframe/algebra/fold.py

```diff
@@ -20,29 +20,26 @@ class Fold(Operator):
     """Builder class for Fold functions."""
 
     @classmethod
-    def call(cls, fold_function, axis=None):
+    def call(cls, fold_function):
         """
         Build Fold operator that will be performed across rows/columns.
 
         Parameters
         ----------
         fold_function : callable(pandas.DataFrame) -> pandas.DataFrame
             Function to apply across rows/columns.
-        axis : int, optional
-            Specifies axis to apply function along.
 
         Returns
         -------
         callable
             Function that takes query compiler and executes Fold function.
         """
 
-        def caller(query_compiler, *args, **kwargs):
+        def caller(query_compiler, fold_axis=None, *args, **kwargs):
```

Comment:
Let's add a doc for the new parameter.

Reply:
This inner function didn't have a docstring at all, but I added it, and documented the new parameter.

Reply:
Done.